pkg/master/master.go

```diff
@@ -544,7 +554,34 @@ func (m *Master) init(c *Config) {
 	}
 
 	// TODO: Attempt clean shutdown?
-	m.masterServices.Start()
+	if m.enableCoreControllers {
+		m.NewBootstrapController().Start()
+	}
+}
+
+// NewBootstrapController returns a controller for watching the core capabilities of the master.
+func (m *Master) NewBootstrapController() *Controller {
```

Comment:
Daniel mentioned that this came up before - if you want to ensure these symbols stay exported, you should leave a comment here and should probably have a test that checks it.  tests/openshift/symbols_test.go ?